process/transaction/interceptedTransaction.go

```diff
@@ -318,7 +318,7 @@ func (inTx *InterceptedTransaction) SenderAddress() []byte {
 
 // Fee returns the estimated cost of the transaction
 func (inTx *InterceptedTransaction) Fee() *big.Int {
-	return inTx.feeHandler.ComputeFee(inTx.tx)
+	return core.SafeMul(inTx.tx.GasPrice, inTx.tx.GasLimit)
```

Comment:
The same comment as above. Is this calculation OK for any case and any call of this method?

Reply:
now it is ok. until now it was wrong.

Reply:
ok